tests/ui/traits/next-solver/diagnostics/projection-trait-ref.stderr

```diff
@@ -0,0 +1,26 @@
+error[E0277]: the trait bound `T: Trait` is not satisfied
```

Comment:
I'm so dumb -- this doesn't actually even test that this PR does anything :(

The error comes from `i32: Trait` WF obligation from `<i32 as Trait>::Assoc`.

Reply:
this test also doesn't enable `-Znext-solver`